### PR TITLE
Fixes #37629 - Support zeitwerk loader

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

**Comment by ShimShtein** [comment]
@ofedoren Can you take a look?

**Review by ofedoren — APPROVED** [review]
Thanks, @ShimShtein, LGTM.

Note for maintainer: ensure that changes in workflow file are removed before merging.

**Review by ekohl — APPROVED** [review]
I dropped the temporary bits.

**Pull request merged**
